index.html

```diff
@@ -13,6 +13,11 @@
     <nav class="mobile-nav-trigger d-flex flex-center">
```

Comment:
Please add files and folders like `.vscode` to `.gitignore`.

Reply:
Hello @SajjadAhmad14, 

> Thank you a lot for this insightful review. I have managed to do as guided and made the push.

_HappyCoding_